pkg/operator/v410_00_assets/bindata.go

```diff
@@ -179,49 +251,16 @@ auditConfig:
       - "RequestReceived"
 authConfig:
   oauthMetadataFile: ""
-  requestHeader:
-    clientCA: /etc/kubernetes/static-pod-certs/configmaps/aggregator-client-ca/ca-bundle.crt
-    clientCommonNames:
-    - kube-apiserver-proxy
-    - system:kube-apiserver-proxy
-    - system:openshift-aggregator
-    extraHeaderPrefixes:
-    - X-Remote-Extra-
-    groupHeaders:
-    - X-Remote-Group
-    usernameHeaders:
-    - X-Remote-User
-  webhookTokenAuthenticators: null
 consolePublicURL: ""
-kubeletClientInfo:
-  ca: /etc/kubernetes/static-pod-resources/configmaps/kubelet-serving-ca/ca-bundle.crt
-  certFile: /etc/kubernetes/static-pod-resources/secrets/kubelet-client/tls.crt
-  keyFile: /etc/kubernetes/static-pod-resources/secrets/kubelet-client/tls.key
-  port: 10250
```

Comment:
this was removed, I think it's okay because it's the default port anyway.

Reply:
> this was removed, I think it's okay because it's the default port anyway.

and there's no obvious flag for me to set.